version

```diff
@@ -1 +1 @@
-1.1.5
+1.1.6
```

Comment:
This is a non-backwards compatible change and therefore the version should be bumped to 2.0.0

Reply:
Will do